src/common/lib/DevTools.browser.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {createDevTools} from 'redux-devtools';
```

Comment:
"Yeah, imports should be sorted."

FWIW, I much prefer to keep following order: node_modules imports, and then este imports, and then my custom imports.

Also, sorting attributes on JSX elements is a real pain, especially when copying over some designers HTML version.